libs/sdk-ui/src/base/measureTitles/ignoreTitlesForSimpleMeasures.ts

```diff
@@ -0,0 +1,75 @@
+// (C) 2007-2020 GoodData Corporation
+import {
+    IInsightDefinition,
+    IBucket,
+    IAttributeOrMeasure,
+    isMeasure,
+    IMeasure,
+    IMeasureDefinitionType,
+    insightBuckets,
+    bucketItems,
+    isMeasureDefinition,
+    isArithmeticMeasureDefinition,
+    isPoPMeasureDefinition,
+    isPreviousPeriodMeasureDefinition,
+} from "@gooddata/sdk-model";
+
+function isAdhocMeasure(definition: IMeasureDefinitionType): boolean {
+    return (
+        isMeasureDefinition(definition) &&
+        (!!definition.measureDefinition.aggregation ||
+            !!definition.measureDefinition.computeRatio ||
+            (Array.isArray(definition.measureDefinition.filters) &&
+                definition.measureDefinition.filters.length > 0))
+    );
+}
```

Comment:
better place for this is the sdk-model package. we should have an exported, specialized type guard for this.

see `isSimpleMeasure` in sdk-model. we can have isAdhocMeasure. export, document, add to barrel. then use here

Reply:
Thanks, I've updated the PR.